src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/MetadataDecoder.vb

```diff
@@ -102,11 +103,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
         End Function
 
         Protected Overrides Function GetTypeHandleToTypeMap() As ConcurrentDictionary(Of TypeDefinitionHandle, TypeSymbol)
-            Return moduleSymbol.TypeHandleToTypeMap
+            Return ModuleSymbol.TypeHandleToTypeMap
```

Comment:
The casing fix here the IDE did as a part of adding the using; it appears correct. This is broken into a separate commit.